packages/apt/test.sh

```diff
@@ -20,7 +20,9 @@ esac
 architecture=$(dpkg --print-architecture)
 
 case "${distribution}-${code_name}" in
-  debian-bookworm|ubuntu-focal)
+  # TODO: We will enable this conditon about Ubuntu focal when we add the case
+  # which is built with mirrored Apache Arrow.
+  debian-bookworm)
```

Comment:
We have already mirrored Apache Arrow for Ubuntu focal.
See: https://packages.groonga.org/ubuntu/pool/focal/main/a/apache-arrow/

So, we should add `ubuntu-focal` into this condition.

Reply:
Thanks. When we build Groonga with Apache Arrow in the Apache Arrow APT Repository, it will fail as follows.
So, if we want to add this condition, I thought we would have to add a pattern that we would build packages with mirrored Apache Arrow.

```
The following packages have unmet dependencies:
 libgroonga0 : Depends: libarrow1900 (>= 19.0.0) but it is not installable
E: Unable to correct problems, you have held broken packages.
```
ref: https://github.com/otegami/groonga/actions/runs/13235719431/job/36940316136#step:14:650

Reply:
Thanks, I see. I will drop support for Ubuntu 20.04 and then restart to do this PR.

Reply:
fix: https://github.com/groonga/groonga/pull/2218 I dropped the support for Ubuntu 20.04.